web/src/scss/components/_modal.scss

```diff
@@ -1,273 +1,250 @@
-.modal-background-overlay{
```

Comment:
See https://github.com/electricitymap/electricitymap-contrib/pull/4228/commits/ea0faf756e6686e2c2e79373a46ad2b9f8b09e90 for the actual changes :)